deps/chakrashim/src/v8proxy.cc

```diff
@@ -22,15 +22,23 @@
 
 namespace v8 {
 Local<Object> Proxy::GetTarget() {
-  // CHAKRA-TODO: Need to add JSRT API to get target of proxy
-  JsValueRef undefinedValue = jsrt::GetUndefined();
-  return Local<Object>::New(undefinedValue);
+  JsValueRef returnValue;
+  bool isProxy = false;
+  JsGetProxyProperties((JsValueRef) this, &isProxy, &returnValue, nullptr );
+  if (!isProxy || returnValue == nullptr) {
```

Comment:
Prefer `JS_INVALID_REFERENCE` over `nullptr` even though they are the same for practical purposes.